rustls-platform-verifier/src/verification/apple.rs

```diff
@@ -183,36 +183,26 @@ impl Verifier {
 impl ServerCertVerifier for Verifier {
     fn verify_server_cert(
         &self,
-        end_entity: &rustls::Certificate,
-        intermediates: &[rustls::Certificate],
-        server_name: &rustls::ServerName,
-        _scts: &mut dyn Iterator<Item = &[u8]>,
```

Comment:
This is somewhat of a downside since Apple [does provide a function for this](https://developer.apple.com/documentation/security/3081274-sectrustsetsignedcertificatetime?language=objc), but then again Apple is the only platform that supports them.

Reply:
I think this would have to be addressed upstream in Rustls.